### ISSUE TITLE
Markdown WYSIWYG Editor is not accessible in High Contrast Mode

### ISSUE THREAD

**Issue by vritant24** [microsoft/vscode#321623]
Steps to Reproduce:

1. Set theme to Light/Dark High Contrast 
2. Open an MD file with WYSIWYG editor

<img width="4590" height="4210" alt="Image" src="https://github.com/user-attachments/assets/fe32d88e-27bb-48e6-a600-0e857b803326" />

<img width="5158" height="4214" alt="Image" src="https://github.com/user-attachments/assets/c58d3c8e-721e-4a4a-be07-646936e1567a" />

**Comment by dozer47528** [comment]
same problem
